process_manifest/iiifManifest.py

```diff
@@ -20,44 +18,59 @@ def manifest(self):
             'label': self._lang_wrapper(self.data.get('title')),
             'thumbnail': self.thumbnail(),
             'items': self._items(),
-            'viewingDirection': 'left-to-right',
-            'seeAlso': [{
-                "id": self._schema_url(),
-                "type": "Dataset",
-                "format": "application/ld+json",
-                "profile": "https://schema.org/"
-            }]
+            'viewingDirection': 'left-to-right'
         }
 
-        if self.data.get('repository', False):
+        if self.key_exists('repository'):
             ret['provider'] = self._add_provider(self.data.repository())
 
-        if self.data.get('usage', False):
+        if self.key_exists('usage'):
             ret['requiredStatement'] = self._convert_label_value('Copyright', self.data.get('usage'))
 
-        if self.data .get('license', False):
+        if self.key_exists('license'):
             ret['rights'] = self.data.get('license')
 
-        if self.data.get('description', False):
+        if self.key_exists('description'):
             ret['summary'] = self._lang_wrapper(self.data.get('description'))
 
-        if False:
+        if self.key_exists('height') and self.key_exists('width') and self.type == 'Canvas':
+            ret['height'] = self.data.get('height')
+            ret['width'] = self.data.get('width')
+
+        if self.key_exists('metsUri'):
+            if 'seeAlso' not in ret:
+                ret['seeAlso'] = []
+
             ret['seeAlso'].append({
-                "id": self.config['manifest-server-base-url'] + '/' + self.id + '/mets.xml',
+                "id": self.data.get('metsUri'),
                 "type": "Dataset",
                 "format": "application/xml",
                 "profile": "http://www.loc.gov/METS/"
             })
 
+        if self.key_exists('imageUri'):
+            if 'seeAlso' not in ret:
+                ret['seeAlso'] = []
+
+            ret['seeAlso'].append({
+                "id": self.data.get('schemaUri'),
+                "type": "Dataset",
+                "format": "application/ld+json",
+                "profile": "https://schema.org/"
+            })
+
         metadata = self.metadata_array()
         if len(metadata) > 0:
             ret['metadata'] = self.metadata_array()
 
         return ret
 
+    def key_exists(self, key):
+        return self.data.get(key, False) and self.data.get(key)
```

Comment:
yes but it needs the second clause.  because in some cases there is an empty key in the csv.  In this case, height and width are examples.  It could be
`
return key in self.data and self.date.get(key)
`